index.html

```diff
@@ -307,7 +355,7 @@ <h3 class="contact-title">Contact me</h3>
           </li>
           <li>
               <label for="msg"></label>
-              <textarea id="msg" name="user_message" maxlength="500" cols="30" rows="10" required placeholder="Write your message here"></textarea>
+              <textarea class="text-area" name="message" maxlength="500" cols="30" rows="10" required placeholder="Write your message here"></textarea>
         </ul>
```

Comment:
Your contact section is almost finished, but you still have some small issues and you can fix it by following the Figma template.

1. Please make sure to fix the size of the top-left border-radius.
2. Kindly make sure to increase the form width a little bit.

![image](https://user-images.githubusercontent.com/34779928/194650328-8cd99947-168f-46d4-9daf-f73de1543cbb.png)